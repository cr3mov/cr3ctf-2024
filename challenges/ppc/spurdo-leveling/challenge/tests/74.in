161275680645674564 949452736257989993
247986 57425
414 94533
