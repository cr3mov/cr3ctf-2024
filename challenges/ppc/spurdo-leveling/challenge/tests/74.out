2268910195611136 15579964262 8304120952144
